tests/integration-all/cognito-user-pool/service/utils.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This file looks as copy of https://github.com/serverless/serverless/blob/master/tests/integration-all/s3/service/utils.js

Maybe it'll be good to find a way to reuse same file (and just ensure it in all tested services)

Reply:
👍 will do